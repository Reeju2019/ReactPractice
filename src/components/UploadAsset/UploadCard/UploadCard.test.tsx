// import { shallow } from "enzyme"
// import ReactDOM from "react-dom"
// import UploadCard from "./UploadCard"

// const localeConfig = {
//   uploadCard_header: "File Name",
//   uploadCard_Size: 12345556,
//   uploadCard_subHeader: "Supported File Formats",
// }

// describe("UploadCard Testing", () => {
//   it("render without crashing", () => {
//     const div = document.createElement("div")
//     ReactDOM.render(
//       <UploadCard
//         uploadedFileName={localeConfig.uploadCard_header}
//         uploadedFIleSize={localeConfig.uploadCard_Size}
//         uploadedFileType={localeConfig.uploadCard_subHeader}
//       />,
//       div
//     )
//     ReactDOM.unmountComponentAtNode(div)
//   })
//   it("should render initial layout", () => {
//     const component = shallow(
//       <UploadCard
//         uploadedFileName={localeConfig.uploadCard_header}
//         uploadedFIleSize={localeConfig.uploadCard_Size}
//         uploadedFileType={localeConfig.uploadCard_subHeader}
//       />
//     )
//     expect(component.getElements()).toMatchSnapshot()
//   })
// })
