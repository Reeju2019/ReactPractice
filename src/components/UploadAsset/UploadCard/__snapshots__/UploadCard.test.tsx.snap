// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UploadCard Testing should render initial layout 1`] = `
Array [
  <React.Fragment>
    <Col
      className="show-drop-images"
      lg={4}
      style={
        Object {
          "flexDirection": "column",
        }
      }
      xs={12}
    >
      <FormLabel
        className="me-2"
        column={false}
        visuallyHidden={false}
      >
        <Card
          body={false}
          className="uploadCard shadow-sm ps-2 pe-4 bg-white rounded"
        >
          <CardBody
            className="m-0 p-0"
          >
            <Row>
              <Col>
                <CardTitle
                  className="uploadCardTitle mt-2"
                >
                  File Name
                </CardTitle>
              </Col>
              <Col>
                <CloseButton
                  aria-label="Close"
                  className="uploadCardClose rounded-circle"
                  id="cancelFileSelection"
                />
              </Col>
            </Row>
            <CardSubtitle
              className="mb-2 text-muted"
              style={
                Object {
                  "fontSize": "10px",
                }
              }
            >
              Supported File Formats
              <br />
              11.77 mb
            </CardSubtitle>
          </CardBody>
        </Card>
      </FormLabel>
    </Col>
  </React.Fragment>,
]
`;
